src/components/SearchPage/SearchPage.jsx

```diff
@@ -17,7 +19,12 @@ const SearchPage = () => {
       headers: {
         Authorization: API_KEY,
       },
-    });
+
+    })
+
```

Comment:
There is too much whitespace between the closing of the block and the starting of `.then` close the gap